src/modules/fortune/containers/FortuneWidget/index.tsx

```diff
@@ -296,7 +297,14 @@ export function FortuneWidget() {
   const [isFortunedShared, setIsFortunedShared] = useState(false)
 
   const [today, setToday] = useState(new Date())
-  useInterval(() => setToday(new Date()), 2_000)
+  const updateToday = useCallback(() => setToday(new Date()), [])
+  useInterval({
+    callback: updateToday,
+    name: 'FortuneWidget',
+    delay: ms`10m`,
```

Comment:
Moving update from 2 seconds to 10min intentional?

Reply:
I should have mention it. I was seeing a lot of checks on this, but i didn't see what is the point on doing this so frequently. 

@shoom3301 is not 10min good enough?